RELEASE/scripts/autoscend/auto_quest_level_8.ash

```diff
@@ -321,7 +321,7 @@ boolean L8_trapperGroar()
 	//What is our potential +Combat score.
 	//TODO: Use that instead of the Avatar/Hound Dog checks.
 
-	if(!canGroar && in_hardcore() && ((auto_my_path() == "Avatar of Sneaky Pete") || !auto_have_familiar($familiar[Jumpsuited Hound Dog]) || forbidFamChange($familiar[Jumpsuited Hound Dog])))
+	if(!canGroar && in_hardcore() && ((auto_my_path() == "Avatar of Sneaky Pete") || !auto_have_familiar($familiar[Jumpsuited Hound Dog]) || !canChangeToFamiliar($familiar[Jumpsuited Hound Dog])))
```

Comment:
This needs rewritten (as it's dumb and doesn't actually do what it's supposed to do which is check if we can't get +combat) and I'm working on it so just ignore it but again the `!auto_have_familiar($familiar[Jumpsuited Hound Dog])` part of the condition could be removed.

Reply:
commit 58b61fc